uploader.json

```diff
@@ -7413,6 +7419,16 @@
                 }
             ]
         },
+        "teamdctl_config_dump": {
```

Comment:
The content in `specs` is deprecated, so this item is not needed.

Reply:
@xiangce - Acknowledge, updated the code as per your suggestion